File: packages/java/src/main/java/com/tesote/sdk/errors/NotFoundException.java
package com.tesote.sdk.errors;

/**
 * Resource-not-found family. Concrete subclasses correspond to specific
 * {@code *_NOT_FOUND} error codes; callers may catch this base when the
 * specific resource doesn't matter.
 */
public class NotFoundException extends ApiException {
    public NotFoundException(String message, String errorCode, int httpStatus,
                             String requestId, String errorId, Integer retryAfter,
                             String responseBody, RequestSummary requestSummary,
                             int attempts, Throwable cause) {
        super(message, errorCode, httpStatus, requestId, errorId,
                retryAfter, responseBody, requestSummary, attempts, cause);
    }
}

File: packages/csharp/src/Errors/NotFoundException.cs
using System;

namespace Tesote.Sdk.Errors;

/// <summary>404 family — resource not found. Concrete subclasses identify which resource.</summary>
public class NotFoundException : ApiException
{
    /// <summary>Construct with the full required-field set.</summary>
    public NotFoundException(
        string? message, string? errorCode, int httpStatus,
        string? requestId, string? errorId, int? retryAfter,
        string? responseBody, RequestSummary? requestSummary,
        int attempts, Exception? cause)
        : base(message, errorCode, httpStatus, requestId, errorId,
               retryAfter, responseBody, requestSummary, attempts, cause)
    {
    }
}
